<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>677</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>74</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>98</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>40</y>
          <width>113</width>
          <height>27</height>
         </rect>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>98</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>10</y>
          <width>61</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>8888</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>10</y>
          <width>113</width>
          <height>27</height>
         </rect>
        </property>
        <property name="inputMask">
         <string>&gt;990.990.990.999</string>
        </property>
        <property name="text">
         <string>127.0.0.1</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Отправленные сообщения</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>Очистить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTreeWidget" name="treeWidget">
           <column>
            <property name="text">
             <string>№</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Время</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Сообщение</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Принятые сообщения</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>Очистить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTreeWidget" name="treeWidget_2">
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>Параметр</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Значение</string>
        </property>
       </column>
      </widget>
      <widget class="QTableWidget" name="tableWidget_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>609</width>
     <height>25</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Команды</string>
    </property>
    <addaction name="action_1"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
    <addaction name="separator"/>
    <addaction name="action_10"/>
    <addaction name="action_11"/>
    <addaction name="action_12"/>
    <addaction name="action_13"/>
    <addaction name="action_14"/>
    <addaction name="action_15"/>
    <addaction name="action_16"/>
    <addaction name="action_17"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Настройки</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>О программе</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <addaction name="menu_4"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_1">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Команда № 1 «Запуск технических средств ОРМ»</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Команда № 2 «Остановка технических средств ОРМ»</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Команда № 3 «Задание пароля»</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Команда № 4 «Закрепление КСЛ за группой»</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Команда № 5 «Постановка объекта на контроль»</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Команда № 6 «Снятие объекта с контроля»</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Команда № 7 «Подключение к разговорному тракту»</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Команда № 8 «Освобождение КСЛ»</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Команда № 9 «Исключение КСЛ из группы»</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>Команда № 10 «Запрос на передачу данных об объектах контроля»</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Команда № 11 «Запрос на передачу информации о соответствии между КСЛ и группами»</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>Команда № 12 «Запрос на передачу списка услуг связи»</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>Команда № 13 «Прерывание выдачи сообщений на запросы содержимого таблиц»</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>Команда № 14 «Тестирование каналов передачи данных»</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>Команда № 15 «Изменение параметров объекта контроля»</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>Команда № 16 «Запрос соответствия имени пучка каналов и его условного номера»</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>Команда № 17 «Запрос версии ПО узла связи»</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
